<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="printButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image...</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Main Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="processedImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>Processed Image</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>421</width>
      <height>611</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Processed Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="imagePathTxt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="medianBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="medianValBox">
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>60</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="minFilterBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Filter</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minfilterInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="maxfilterInput">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="maxFilterBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNoiseBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Noise</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="noiseInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="reobertBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reobert Borders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SobelBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AverageBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Average</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="horiaontalLaplace">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="laplicianBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Laplacien</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="verticalLaplace">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButterLowBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ButterLow</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="butterOrderInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>n Order</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="butterDoinput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Do</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vguassianInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Image V</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Image U</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GaussianBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="uguassianInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="foureirBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fourier</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="magnitudeInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Magnitude Spectrum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Level</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="grayInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>256.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="negativeBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ThresholdingBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thresholding</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Transform</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="powerInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="powerBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PowerLaw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contrastBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast stretching</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="contrastr2">
    <property name="geometry">
     <rect>
      <x>667</x>
      <y>470</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="contrastr1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HistgoramBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Histgoram</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="widthInput">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1080.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resizeBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="heightInput">
    <property name="geometry">
     <rect>
      <x>667</x>
      <y>520</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1080.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>printButton</zorder>
   <zorder>mainImage</zorder>
   <zorder>label</zorder>
   <zorder>processedImage</zorder>
   <zorder>imagePathTxt</zorder>
   <zorder>medianBtn</zorder>
   <zorder>medianValBox</zorder>
   <zorder>minFilterBtn</zorder>
   <zorder>minfilterInput</zorder>
   <zorder>maxfilterInput</zorder>
   <zorder>maxFilterBtn</zorder>
   <zorder>addNoiseBtn</zorder>
   <zorder>noiseInput</zorder>
   <zorder>reobertBtn</zorder>
   <zorder>SobelBtn</zorder>
   <zorder>AverageBtn</zorder>
   <zorder>horiaontalLaplace</zorder>
   <zorder>laplicianBtn</zorder>
   <zorder>label_3</zorder>
   <zorder>verticalLaplace</zorder>
   <zorder>label_4</zorder>
   <zorder>ButterLowBtn</zorder>
   <zorder>butterOrderInput</zorder>
   <zorder>label_5</zorder>
   <zorder>butterDoinput</zorder>
   <zorder>label_6</zorder>
   <zorder>vguassianInput</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>GaussianBtn</zorder>
   <zorder>uguassianInput</zorder>
   <zorder>foureirBtn</zorder>
   <zorder>magnitudeInput</zorder>
   <zorder>label_9</zorder>
   <zorder>grayBtn</zorder>
   <zorder>grayInput</zorder>
   <zorder>negativeBtn</zorder>
   <zorder>ThresholdingBtn</zorder>
   <zorder>logBtn</zorder>
   <zorder>powerInput</zorder>
   <zorder>powerBtn</zorder>
   <zorder>contrastBtn</zorder>
   <zorder>contrastr2</zorder>
   <zorder>contrastr1</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>HistgoramBtn</zorder>
   <zorder>widthInput</zorder>
   <zorder>label_12</zorder>
   <zorder>resizeBtn</zorder>
   <zorder>heightInput</zorder>
   <zorder>label_13</zorder>
   <zorder>saveBtn</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>printButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
